Index: src/main/java/com/pdramirez/restservice/LoadDatabase.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/pdramirez/restservice/LoadDatabase.java b/src/main/java/com/pdramirez/restservice/LoadDatabase.java
--- a/src/main/java/com/pdramirez/restservice/LoadDatabase.java	
+++ b/src/main/java/com/pdramirez/restservice/LoadDatabase.java	(date 1675978732629)
@@ -2,6 +2,9 @@
 
 import com.pdramirez.restservice.employee.Employee;
 import com.pdramirez.restservice.employee.EmployeeRepository;
+import com.pdramirez.restservice.order.Order;
+import com.pdramirez.restservice.order.OrderRepository;
+import com.pdramirez.restservice.order.Status;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.boot.CommandLineRunner;
@@ -11,12 +14,17 @@
 @Configuration
 public class LoadDatabase {
     private static final Logger log = LoggerFactory.getLogger(LoadDatabase.class);
+
     @Bean
-    CommandLineRunner initDatabase(EmployeeRepository employeerepository){
+    CommandLineRunner initDatabase(EmployeeRepository employeerepository, OrderRepository orderRepository) {
 
-        return  args -> {
-            log.info("Preloading "+ employeerepository.save(new Employee("Bilbo Baggins","burglar")));
-            log.info("Preloading "+employeerepository.save(new Employee("Frodo Baggins","thief")));
+        return args -> {
+            employeerepository.save(new Employee("Pepe", "Cabeza", "Hero"));
+            employeerepository.save(new Employee("Pipo", "Perez", "rno"));
+
+            employeerepository.findAll().forEach(employee -> {
+                log.info("Preoloaded "+employee);
+            });
             orderRepository.save(new Order("MacBook Pro", Status.COMPLETED));
             orderRepository.save(new Order("iPhone", Status.IN_PROGRESS));
 
